### PR TITLE
Add Django 1.10 support

### PR DESCRIPTION
**Pull request by Ian-Foote**
Builds on a rebased version of #81.

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -1,13 +1,12 @@
...
+coverage==4.2
+dj-database-url==0.4.1
+django~=1.10.4
+factory_boy==2.8.1
+flake8==3.2.1
+flake8-docstrings==1.0.2
+flake8-import-order==0.11
+incuna-test-utils==6.6.0
+mkdocs==0.16.0
+psycopg2==2.6.2
```
Comment (Ian-Foote) [inline requirements.txt]: Do you want to use `requires.io` to manage these?

Comment (meshy) [inline requirements.txt]: Good idea, but not sure that I can deal with the emails for the moment -- perhaps we should add a badge. I'll do it in another PR.

**Comment by meshy** [comment]
Thanks for finishing this! I'll give it a look `:)`

File: conman/routes/tests/urls.py [hunk 2]
```diff
@@ -1,14 +1,12 @@
...
-
+from unittest import mock
 
-def dummy_view():
-    """A stub view that is always mocked."""
+from django.conf.urls import url
 
 
-view_path = 'conman.routes.tests.urls.dummy_view'
+dummy_view = mock.MagicMock()
```
Comment (meshy) [inline conman/routes/tests/urls.py]: Smart! I didn't think of this.

**Pull request merged**